docs/src/components/header/HeaderNavDropdown.tsx

```diff
@@ -67,6 +68,8 @@ const PRODUCTS = [
 ];
 
 export default function HeaderNavDropdown() {
+  const router = useRouter();
+  const asPathWithoutLang = router.asPath.replace(/^\/[a-zA-Z]{2}\//, '/');
```

Comment:
We could have used https://github.com/mui/material-ui/blob/7310d056b916bb248b7dfee790b2de1776349510/docs/src/modules/utils/helpers.ts#L257